cypress/TestCases/FireboltCertification/Manage/HdmiInput.feature

```diff
@@ -0,0 +1,150 @@
+Feature: HdmiInput_Manage
+
+   Background: Initialize 'HdmiInput'
+      Given the environment has been set up for 'hdmiInput' tests
+      And 3rd party 'certification' app is launched
+
+   @HdmiInput @manageSDK @sdk @transport
+   Scenario: HDMIInput.ports - Positive Scenario: Validate HDMIInput ports
+      When 1st party app invokes the 'Firebolt' API to 'get hdmiinput ports list'
+      Then 'Firebolt' platform responds to '1st party app' with 'expected hdmiinput ports'
+
+   @HdmiInput @manageSDK @sdk @transport
+   Scenario Outline: HDMIInput.port - Positive Scenario: <Scenario>
+      When 1st party app invokes the 'Firebolt' API to '<Key>'
+      Then 'Firebolt' platform responds to '1st party app' with 'expected hdmiinput port'
+
+      Examples:
+         | Scenario                        | Key                                  |
+         | Validate HDMIInput port - Hdmi1 | get hdmiinput port with portId HDMI1 |
+         | Validate HDMIInput port - Hdmi2 | get hdmiinput port with portId HDMI2 |
+
+   @HdmiInput @manageSDK @sdk @transport
+   Scenario Outline: HDMIInput.port - Positive Scenario: <Scenario>
+      When 1st party app invokes the 'Firebolt' API to '<Key>'
+      When 1st party app invokes the 'Firebolt' API to 'get hdmiinput port with portId HDMI1'
+      Then 'Firebolt' platform responds to '1st party app' with '<Validation_Key>'
+
```

Comment:
setter validation also to be added after we call setter

Reply:
We don't have setter validation before, Now added setter validation similar to 2044. Pulled 2044 and tested.